src/game_api/script/usertypes/game_manager_lua.cpp

```diff
@@ -78,10 +79,99 @@ void register_usertypes(sol::state& lua)
     gamemanager_type["buttons_controls"] = &GameManager::buttons_controls;
     gamemanager_type["buttons_movement"] = &GameManager::buttons_movement;
 
+    /// Used in GameManager
     lua.new_usertype<SaveRelated>(
         "SaveRelated",
         "journal_popup_ui",
-        &SaveRelated::journal_popup_ui);
+        &SaveRelated::journal_popup_ui,
+        "places_data",
+        &SaveRelated::places_data,
+        "bestiary_data",
+        &SaveRelated::bestiary_data,
+        "monster_part_to_main",
+        &SaveRelated::monster_part_to_main,
+        "people_info",
+        &SaveRelated::people_info,
+        "people_part_to_main",
+        &SaveRelated::people_part_to_main,
+        "item_info",
+        &SaveRelated::item_info,
+        "trap_info",
+        &SaveRelated::trap_info,
+        "trap_part_to_main",
+        &SaveRelated::trap_part_to_main,
+        "stickers_data",
+        &SaveRelated::stickers_data,
```

Comment:
So i only tested inserting new record, which works xd
you can get element with `:at(index)`, but that's probably uses the same method as `pairs`, I'm surprised that `:find(key)` doesn't work. It seams to treat it as array instead of table 🤔 

Wondering if it's related to updating to C++20

std::map works fine (at least, the `state.room_owners.owned_items`

Reply:
wait, the map in EntityDB works fine?

Reply:
ok, it needs hasher to be defined